src/rom_header_gf.c

```diff
@@ -108,7 +108,6 @@ static const struct GFRomHeader sGFRomHeader = {
     //.monIconPaletteIds = gMonIconPaletteIndices,
     .monIconPalettes = gMonIconPaletteTable,
     //.monSpeciesNames = gSpeciesNames, // Handled in gSpeciesInfo
-    .moveNames = gMoveNames,
```

Comment:
We just comment out now unused fields rather than outright removing them (see other comments in this file)

Reply:
Got it, I'll re-add it and leave it commented out then :eyes: